docs/sources/mimir/manage/tools/mimirtool.md

```diff
@@ -217,6 +217,24 @@ The following command verifies if alerts in an Alertmanager cluster are deduplic
 mimirtool alerts verify
 ```
 
+#### Render template
+
+You can render your Alertmanager template with the `render` command.
+
+The argument to the `--template-glob` option is a glob that expands to the alert template paths.
```

Comment:
I think we can use `.` and make it the same as `amtool`:

```suggestion
The argument to the `--template.glob` option is a glob that expands to the alert template paths.
```

Reply:
Oh OK. @56quarters I see mimirtool also uses `.` to denote parent/child relationships, for example:

```
--log.level="info"
--push-gateway.endpoint=PUSH-GATEWAY.ENDPOINT
--push-gateway.job=PUSH-GATEWAY.JOB
--push-gateway.interval=1m
```

Reply:
Yes it was I saw also but for me it’s similar . I am ok with both of flag ^^

Reply:
Sorry, when I said "Could you keep the style of this flag consistent with the other flags?" I was referring the code - not the option names. Let's make the changes @grobinson-grafana suggested

Reply:
Modification made. Is that good for you ?